inc/ThirdParty/Plugins/SimpleCustomCss.php

```diff
@@ -58,33 +53,66 @@ public function cache_sccss() {
 	 * @since 2.9
 	 * @author Remy Perona
 	 */
-	public function delete_cache_file() {
-		$sccss = rocket_get_cache_busting_paths( 'sccss.css', 'css' );
-
-		array_map( 'unlink', glob( $sccss['bustingpath'] . 'sccss*.css' ) );
+	public function update_cache_file() {
```

Comment:
The method `delete_cache_file()` should be deprecated.

Reply:
Originally it was : function rocket_delete_sccss_cache_file(), which is deprecated.
Will update the deprecated to:

Reply:
The method `delete_cache_file()` was public, it still needs to be deprecated.

Reply:
The method `delete_cache_file()` was not in any production code, only in this PR. 
So I don't think it should be deprecated because I simply renamed it.
Opinions about this one?

Reply:
I don't understand, it is tagged `@since 2.9`.

Reply:
Yes, because it was this function before: https://github.com/wp-media/wp-rocket/blob/927900bfdd59d25a6809eaa358b2357886af0220/inc/deprecated/3.6.php#L74

Reply:
OK, then the `@since` tag is misleading, it should be `3.6` imho, as this method doesn't exist before 3.6.